src/FSharpVSPowerTools.Logic/ImplementInterfaceSmartTagger.fs

```diff
@@ -24,18 +24,21 @@ type InterfaceState =
       EndPosOfWith: pos option
       Tokens: FSharpTokenInfo list }
 
-type ImplementInterfaceSmartTagger(textDocument: ITextDocument,
-                                   view: ITextView, 
-                                   editorOptionsFactory: IEditorOptionsFactoryService, 
-                                   textUndoHistory: ITextUndoHistory,
-                                   vsLanguageService: VSLanguageService, 
-                                   serviceProvider: IServiceProvider,
-                                   projectFactory: ProjectFactory,
-                                   objectIdentifier: string,
-                                   defaultBody: string) as self =
-    let tagsChanged = Event<_, _>()
+type ImplementInterface
```

Comment:
Could you get rid of these warnings https://ci.appveyor.com/project/dungpa/visualfsharppowertools/build/2.1.0-a1508100652/messages?